deeplearning4j-core/src/test/java/org/deeplearning4j/nn/layers/custom/testclasses/CustomLayer.java

```diff
@@ -79,4 +80,9 @@ public void setNIn(InputType inputType, boolean override) {
     public InputPreProcessor getPreProcessorForInputType(InputType inputType) {
         return null;
     }
+
+    @Override
+    public LayerMemoryReport getMemoryReport(InputType inputType) {
+        throw new UnsupportedOperationException();
+    }
```

Comment:
@raver119 I actually wanted to factor in  our activities with the op executioner in to the execution graph. If we can figure out a way to layer in the execution graph "underneath" or "wrap" the op executioner we can get op execution via graphs for free.